### PR TITLE
Update ansibleVars

### PR DESCRIPTION
**Pull request by fao89**
based on https://github.com/openstack-k8s-operators/install_yamls/pull/156

### CODE REVIEW

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml [hunk 1]
```diff
@@ -55,5 +117,23 @@ spec:
...
+          edpm_hosts_entries_undercloud_hosts_entries: []
+          # edpm_hosts_entries role
+          edpm_hosts_entries_extra_hosts_entries:
+          - 172.17.0.80 glance-internal.openstack.svc neutron-internal.openstack.svc cinder-internal.openstack.svc nova-internal.openstack.svc placement-internal.openstack.svc keystone-internal.openstack.svc
+          - 172.17.0.85 rabbitmq.openstack.svc
+          - 172.17.0.86 rabbitmq-cell1.openstack.svc
+          edpm_hosts_entries_vip_hosts_entries: []
+          hosts_entries: []
+          hosts_entry: []
+          plan: overcloud
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplane.yaml:137]: plan is not needed.

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml [hunk 2]
```diff
@@ -35,17 +43,71 @@ spec:
...
+
+          edpm_ovn_metadata_agent_default_transport_url: rabbit://default_user@rabbitmq.openstack.svc:5672
+          edpm_ovn_metadata_agent_metadata_agent_ovn_ovn_sb_connection: tcp:10.217.5.121:6642
+          edpm_ovn_metadata_agent_metadata_agent_default_nova_metadata_host: 127.0.0.1
+          edpm_ovn_metadata_agent_metadata_agent_default_metadata_proxy_shared_secret: 12345678
+          edpm_ovn_metadata_agent_default_bind_host: 127.0.0.1
+          edpm_chrony_ntp_servers:
+          - clock.redhat.com
+
+          # 99-standalone-vars
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplane.yaml:104]: drop this comment

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml [hunk 3]
```diff
@@ -35,17 +43,71 @@ spec:
...
+          edpm_nodes_validation_validate_controllers_icmp: false
+          edpm_nodes_validation_validate_gateway_icmp: false
+
+          edpm_ovn_metadata_agent_default_transport_url: rabbit://default_user@rabbitmq.openstack.svc:5672
+          edpm_ovn_metadata_agent_metadata_agent_ovn_ovn_sb_connection: tcp:10.217.5.121:6642
+          edpm_ovn_metadata_agent_metadata_agent_default_nova_metadata_host: 127.0.0.1
+          edpm_ovn_metadata_agent_metadata_agent_default_metadata_proxy_shared_secret: 12345678
+          edpm_ovn_metadata_agent_default_bind_host: 127.0.0.1
+          edpm_chrony_ntp_servers:
+          - clock.redhat.com
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplane.yaml:102]: i would add clock2.redhat.com here as well

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml [hunk 4]
```diff
@@ -35,17 +43,71 @@ spec:
...
+          storage_cidr: '24'
+          storage_host_routes: []
+          tenant_mtu: 1500
+          tenant_vlan_id: 22
+          tenant_cidr: '24'
+          tenant_host_routes: []
+          role_networks:
+          - InternalApi
+          - Storage
+          - Tenant
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplane.yaml:85]: I think we might need External here

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml [hunk 5]
```diff
@@ -35,17 +43,71 @@ spec:
...
         ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
         ansibleVars: |
+          service_net_map:
+            nova_api_network: internal_api
+            nova_libvirt_network: internal_api
+
+          # edpm_network_config
+          # Default nic config template for a EDPM compute node
+          # These vars are edpm_network_config role vars
           edpm_network_config_template: templates/net_config_bridge.j2
```
Comment (slagle) [inline config/samples/dataplane_v1beta1_openstackdataplane.yaml:53]: this should be the default value from install_yamls shouldn't it?
`templates/single_nic_vlans/single_nic_vlans.j2`

**Review by slagle — CHANGES_REQUESTED** [review]

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml (hunk 1 above)
Comment (fao89) [reply config/samples/dataplane_v1beta1_openstackdataplane.yaml]: done!

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml (hunk 4 above)
Comment (fao89) [reply config/samples/dataplane_v1beta1_openstackdataplane.yaml:85]: done!

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml (hunk 5 above)
Comment (fao89) [reply config/samples/dataplane_v1beta1_openstackdataplane.yaml]: done!

**Review by slagle — APPROVED** [review]

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml (hunk 1 above)
Comment (fao89) [reply config/samples/dataplane_v1beta1_openstackdataplane.yaml]: actually, it was needed: https://github.com/openstack-k8s-operators/edpm-ansible/pull/80

File: config/samples/dataplane_v1beta1_openstackdataplane.yaml (hunk 4 above)
Comment (fao89) [reply config/samples/dataplane_v1beta1_openstackdataplane.yaml:85]: https://github.com/openstack-k8s-operators/install_yamls/pull/172

**Pull request merged**